src/App.tsx

```diff
@@ -1,7 +1,35 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  pressedKey: string,
+};
+
+export class App extends Component<{}, State> {
+  state: Readonly<State> = {
+    pressedKey: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.handleKeyup);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.handleKeyup);
+  }
+
+  handleKeyup = (event: KeyboardEvent) => {
+    this.setState({ pressedKey: event.key });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <p className="App__message">
+          {!this.state.pressedKey
+            ? 'Nothing was pressed yet'
+            : `The last pressed key is [${this.state.pressedKey}]`}
```

Comment:
This pressedKey value was used twice, so let's destructure it and create a variable

```suggestion
  render() {
    const pressedKey = this.state.pressedKey;
    
    return (
      <div className="App">
        <p className="App__message">
          {!pressedKey
            ? 'Nothing was pressed yet'
            : `The last pressed key is [${pressedKey}]`}
```